### ISSUE TITLE
Is there anyway that I can use learning rate warm-up during the training ? 

### ISSUE THREAD

**Issue by shamanez** [huggingface/alignment-handbook#92]
I am using this repo to:
1. Continual Pre-training 
2. SFT
3. DPR

For stage 1, I want to use a learning rate warm-up.
